catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/resource/download/ReliableResourceDownloaderTest.java

```diff
@@ -61,6 +65,8 @@
 import ddf.catalog.resource.ResourceNotSupportedException;
 import ddf.catalog.resourceretriever.ResourceRetriever;
 
+@RunWith(PowerMockRunner.class)
```

Comment:
Fixed so that the coverage properly shows up in the report, and refactored the test for better coverage.